api/README.md

```diff
@@ -1,3 +1,10 @@
+# Nessie Specification
+
+The Nessie `ConfigApi` (v2) allows the caller to retrieve the server's 'specVersion', which is a semver string
+defining the behaviour of the Nessie Server.
+
+The currently defined Nessie Specification is [2.0.0-beta.1](NESSIE-SPEC-2-0.md).
```

Comment:
I'm wondering: when the spec changes, we'd need to go through all the docs that reference the current spec version, such as the ones in this PR, and change them. Do we have an automated process in place for that?

Reply:
We have some automation for Nessie version bumps, but it's not perfect either... We had some bugs in docs because of the automation too :sweat_smile: